docs/advanced/accesstransformers.md

```diff
@@ -0,0 +1,111 @@
+Access Transformers
+===================
+
+Access Transformers (ATs for short) allows for changing the visibility and `final` flags of classes and their methods and fields. They allow modders to access and modify otherwise `private` members in classes outside their control.
```

Comment:
My thought was to make the opening paragraph simple to understand. The second sentence states that it allows access to `private` members, because to modders, this is the most (and often the only) usecase they have. The first sentence already covers any visiblity. Though, reading at it again, I might change 'changing' to 'widening', since 'changing' could mean restricting the visibility.

Reply:
I think I'll change it to `otherwise inaccesible`.